.gitignore

```diff
@@ -3,6 +3,7 @@ ChangeLog
 
 #### Python
 *.py[co]
+/venv
```

Comment:
Please drop the changes to this `.gitignore` file, and use a local gitignore instead.

Reply:
Done. Good suggestion.